composer.json

```diff
@@ -14,7 +14,7 @@
         "ext-fileinfo": "*",
         "knplabs/github-api": "^2.11",
         "php-http/guzzle6-adapter": "^1.1",
-        "phptaskman/core": "^0.1.16"
```

Comment:
Hi mate,

The branch 0.x of Taskman is supporting PHP >= 5.6.
The branch 1.x of Taskman is supporting PHP >= 7.1.

There is currently no version of this package for Taskman 1.x. I think the best would be to merge this in master and create a branch 0.x from the previous commit.

What do you think ?

Reply:
I'll merge this and do the branch 0.x right now.

Reply:
That sounds good. I think with PHP 5.6 end of life approaching closer the branches 0.x will not need much more work anyway. But good to have the branch there.

Reply:
It's done! Release 1.0.0 of this has been tagged as well :-)